### ISSUE TITLE
Checkbox and Radio (data-toggle buttons) are not accessible by pressing tab

### ISSUE THREAD

**Issue by pajind** [twbs/bootstrap#26778]
Tested on both edge and chrome.
Checkbox and radio buttons are not accessible when pressing the TAB key and skips it

**Comment by patrickhlauke** [comment]
please provide a reduced test case (in jsfiddle or similar) showing the issue

**Comment by scottblinch** [comment]
You'll actually be able to reproduce this issue in the docs: 

https://getbootstrap.com/docs/4.0/components/buttons/#checkbox-and-radio-buttons

I've also made a pen:

https://codepen.io/scottblinch/pen/PagNmW

Try using just tab to navigate to the second 2nd button in the button group. The arrow keys do work, however, as this is the normal keyboard navigation behaviour for grouped radio buttons.

The problem here might be a perceptual one, and it's something I've noticed has existed since v3. Where a user using a screenreader would get the hint that they've just navigated into a grouped radio button and would know to use arrow keys, a sighted keyboard user would likely assume that they're just on a button/anchor, and then get confused about tab not bringing them to the next in group.

I'm not sure what the solution to this would be, since adding tabbing support to toggle button groups would likely be detrimental to those using screenreaders. (I think -- hopefully someone out there more versed in a11y than I am can chime in.)

**Comment by RinkAttendant6** [comment]
According to one of the ARIA best practices example, the <kbd>Tab</kbd> key should bring the user to the next group. Navigation within the buttons in the group is by arrow keys.

https://w3c.github.io/aria-practices/examples/radio/radio-1/radio-1.html

More information available regarding "roving tabindex" here: https://w3c.github.io/aria-practices/#kbd_roving_tabindex

The current Bootstrap behaviour seems to move between buttons in the radio group when pressing <kbd>Tab</kbd> which is IMHO unexpected and annoying. For a form of 20 questions with 4 options each: it takes x arrows + up to 80 tabs as opposed to x arrows + 20 tabs. Also it doesn't even seem to work correctly in Chrome, Edge, or Firefox due to #26855 (works fine in IE 11).
